Makefile

```diff
@@ -8,11 +8,17 @@ lint:
 	go get github.com/golang/lint/golint
 	golint *.go
 
-test: sysfs/fixtures/.unpacked
+test: sysfs/fixtures/.copied
```

Comment:
Working on it. The straightforward way will require GNU make, is that acceptable?